packages/eslint-plugin-udemy/README.md

```diff
@@ -26,15 +26,30 @@ You can additionally add settings for the plugin.
 
 # Contributing
 
-Contributions are always welcome! For more info, please get in touch with @udemy/team-f team. In a nutshell:
+Contributions are always welcome! The main approach described at 
+[top-level Contributing section](/#contributing) applies to this package, as all packages.
 
-1. Run `npm install` to install all dependencies.
-1. Make your desired changes to any rule under `rules` folder. Or add a new rule under `rules` folder
-following the existing code pattern.
-1. *Do* update `version` number in `package.json` file as necessary.
-1. Run `npm run test` and make sure all tests and lints are passing.
-1. Finally, run `npm publish` to publish your changes to npm.
+### Updating an existing rule
 
-# License
+Just follow the approach described at [top-level Contributing section](/#contributing), and do not forget to update
+both rule-specific `README.md` file and this package's 
+[List of provided rules](#list-of-provided-rules) section.
 
-(c) Copyright 2017 Udemy, Inc., all rights reserved.
+
+### Adding a new rule
+
+This repository has a set of custom ESLint rules implemented under [`rules`](packages/eslint-plugin-udemy/rules)
+folder. In order to create a new rule, you should:
+
+1. Create a new folder under [`rules`](packages/eslint-plugin-udemy/rules) folder with the desired rule name.
+1. Implement the rule at `index.js` file, following existing examples. *You can also refer to ESLint's 
+[Working with Rules](https://eslint.org/docs/developer-guide/working-with-rules) documentation to learn
+more.*
+1. Once the rule is implemented, create `tests.js` file where you will write necessary tests for the rule,
+again following existing examples. *You can refer to ESLint's 
+[RuleTester](https://eslint.org/docs/developer-guide/nodejs-api#ruletester) documentation to learn more.*
+1. Create a `README.md` file that describes the details of the rule, based on 
```

Comment:
based on the

Reply:
If only I had Grammarly installed on my PyCharm IDE. I wonder if there is such a plugin :D

Reply:
100% agreed!